software/webhook-tester.yml

```diff
@@ -0,0 +1,11 @@
+name: "WebHook Tester"
+source_code_url: "https://github.com/tarampampam/webhook-tester"
+description: "Powerful tool for testing WebHooks and more."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Go
+  - deb
```

Comment:
> https://github.com/tarampampam/webhook-tester?tab=readme-ov-file#-installation
```suggestion
platforms:
  - Docker
  - Go
```

Reply:
Even if `.deb` files are available to download?

![image](https://github.com/user-attachments/assets/8f118f50-930e-47d0-bdef-137918810836)

Reply:
Ahh, ignore this suggestion, I didnt see those!